pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
   "parfive[ftp]>=2.0.0",
   "pyerfa>=2.0",
   "requests>=2.28.0",
+  "fsspec==2024.6.1"
```

Comment:
`fsspec` is also called in `sunpy.io._file_tools.py` (used in `detect_filetypes` function) so that's why I put the dependency in core. But if there is a workaround to that, I'm happy to edit.